prettyconf/loaders.py

```diff
@@ -308,6 +312,9 @@ def __getitem__(self, item):
 
 class AwsParameterStore(AbstractConfigurationLoader):
     def __init__(self, path="/", aws_access_key_id=None, aws_secret_access_key=None, region_name="us-east-1"):
+        if not _have_boto:
+            raise RuntimeError("'boto3' package is required by AwsParameterStore loader.")
```

Comment:
Put additional instructions about how to fix it: `Install prettyconf[aws] extra feature.`